openmc/model/funcs.py

```diff
@@ -345,6 +345,54 @@ def get_hexagonal_prism(edge_length=1., orientation='y', origin=(0., 0.),
     return prism
 
 
+def cylinder_from_points(p1, p2, r, **kwargs):
```

Comment:
perhaps 'r' should be 'R' for consistency with openmc.ZCylinder

Reply:
To me, it's actually a mistake that ZCylinder and some other surfaces have capitalized argument names (which defies the style guide), so I didn't want to propagate that mistake elsewhere.

Reply:
I definitely agree with you... but the consistency either way would be nice.